src/components/Common/SortDropdown.tsx

```diff
@@ -22,31 +28,31 @@ interface Props {
 export default function SortDropdownMenu(props: Props) {
   const { t } = useTranslation();
   return (
-    <DropdownMenu
-      title={props.label ?? t("sort_by")}
-      variant="secondary"
-      className="w-full border border-primary-500 bg-white md:w-auto"
-      icon={<CareIcon icon="l-sort" />}
-      containerClassName="w-full md:w-auto z-20"
+    <Select
+      value={props.selected}
+      onValueChange={(v) => props.onSelect({ ordering: v })}
     >
-      {props.options.map(({ isAscending, value }, i) => (
-        <DropdownItem
-          key={i}
-          className={
-            props.selected === value
-              ? "bg-primary-100 !font-medium text-primary-500"
-              : ""
-          }
-          onClick={() => props.onSelect({ ordering: value })}
-          icon={
+      <SelectTrigger className="w-auto">
+        {props.selected ? (
+          <SelectValue />
+        ) : (
+          <div>
+            <CareIcon icon="l-sort" className="mr-2" />
+            {t("sort_by")}
+          </div>
+        )}
+      </SelectTrigger>
+      <SelectContent>
+        {props.options.map(({ isAscending, value }, i) => (
```

Comment:
Yes, it looks quite redundant because the value has a `-` to mark it as descending, but it makes sense for options like 

```
{
    isAscending: true,
    value: "facility__name,last_consultation__current_bed__bed__name",
  },
  {
    isAscending: false,
    value: "facility__name,-last_consultation__current_bed__bed__name",
  },
```